toys/release/.data/rad-yard-templates/default/layout/yaml/_constant.erb

```diff
@@ -5,6 +5,7 @@
   langs:
     - ruby
   module: <%= ENV["CLOUDRAD_GEM_NAME"] %>
+  friendlyApiName: <%= ENV["CLOUDRAD_FRIENDLY_API_NAME"] %>
```

Comment:
Can you do this for each instance of the friendly API name, in this and each of the other erb templates?

```ruby
friendlyApiName: <%= ENV["CLOUDRAD_FRIENDLY_API_NAME"].inspect %>
```

This will cause the value to be enclosed in double quotes (and will escape quotes correspondingly) to handle any case of the friendly name including yaml-unfriendly characters.

Reply:
Done, thanks! I was wondering if there was a way to not include the ```friendlyApiName``` key if there is no value for ```CLOUDRAD_FRIENDLY_API_NAME```? I notice when the env variable is not set, the key gets value ```nil```.

Reply:
Oh, good catch. If you replace `.inspect` with `.to_s.inspect`, that will cause it to be set to the empty string (i.e. `""`) instead of the string `"nil"`. Or, if you want to omit the key altogether, you could:

```ruby
<% if ENV["CLOUDRAD_FRIENDLY_API_NAME"] %>
  friendlyApiName: <%= ENV["CLOUDRAD_FRIENDLY_API_NAME"].inspect %>
<% end %>
```

Reply:
Thanks, made empty it string.